src/components/ProductMilestoneBuildsPerformedPage/ProductMilestoneBuildsPerformedPage.tsx

```diff
@@ -58,5 +61,20 @@ export const ProductMilestoneBuildsPerformedPage = ({ componentId = 'b1' }: IPro
     )
   );
 
-  return <BuildsList {...{ serviceContainerBuilds, componentId }} />;
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem reservedWidth>
+          <TextContent>
+            <Text component={TextVariants.h2}>Builds Performed</Text>
+            <Text>
+              This list contains Builds in the Milestone. Builds produced from Build Configuration are assigned to the current
+              Milestone in Product Version linked in the Configuration.
```

Comment:
`This list contains Builds in the Milestone. Builds produced from the Build Configuration are assigned to the current Milestone in the Product Version linked in the Build Configuration.`